pkg/apis/eventing/v1alpha1/stub_test.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import "testing"
+
+func TestStub(t *testing.T) {
+	// TODO: Implement tests in this directory, please.
+	// This file is used to make code coverage on this directly count.
```

Comment:
Can you add some simple tests for Provisioner (e.g. that `type` is required) and that Defaults "work"?